src/studio/src/designer/frontend/app-development/features/handleMergeConflict/components/HandleMergeConflictDiscardChanges.test.tsx

```diff
@@ -1,202 +1,72 @@
-import { mount } from 'enzyme';
 import React from 'react';
 import * as networking from 'app-shared/utils/networking';
 
 import { HandleMergeConflictDiscardChanges } from './HandleMergeConflictDiscardChanges';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+const consoleError = jest
+  .spyOn(console, 'error')
+  .mockImplementation(() => 'error');
+
+const renderHandleMergeConflictDiscardChanges = () => {
+  const user = userEvent.setup();
+
+  const container = render(
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    <HandleMergeConflictDiscardChanges classes={{}} language={{}} />,
```

Comment:
We really shouldnt be adding this stuff. I assume its because of the classess stuff and the typings?

If thats the case, it can be avoided by instead importing the default export instead of the named export.

```suggestion
    <HandleMergeConflictDiscardChanges classes={{}} language={{}} />,
```